packages/ember/src/components/mktg/footer.gts

```diff
@@ -0,0 +1,80 @@
+import Component from '@glimmer/component';
+import { or } from 'ember-truth-helpers';
+
+interface FooterSectionSignature {
+  Element: HTMLDivElement;
+  Args: {
+    isCollapsible?: boolean;
+  };
+  Blocks: {
+    default?: [];
+  };
+}
+
+interface MarketingFooterSignature {
+  Element: HTMLElement;
+  Args: {
+    hasHorizontalLine?: boolean;
+  };
+  Blocks: {
+    nav: [];
+    'social-media': [];
+    brand: [];
+    legal: [];
+    default?: [];
+  };
+}
+
+class FooterSection extends Component<FooterSectionSignature> {
+  get isCollapsible() {
+    return this.args.isCollapsible ?? true;
+  }
+
+  <template>
+    <div
+      class="col row
+        {{if this.isCollapsible 'row-cols-1 row-cols-md-auto' 'row-cols-auto'}}
+        align-items-center gx-3 gy-4 gx-md-4 my-0"
+    >
+      {{yield}}
+    </div>
+  </template>
+}
+
+export default class MarketingFooterComponent extends Component<MarketingFooterSignature> {
+  get hasHorizontalLine() {
+    return this.args.hasHorizontalLine ?? false;
+  }
+
+  <template>
+    <footer class="bg-primary text-light mt-auto" ...attributes>
+      <div class="container p-5">
+        <div
+          class="row row-cols-1 row-cols-md-auto justify-content-between align-items-center"
+        >
+          <FooterSection>
+            {{yield to="nav"}}
+          </FooterSection>
+          <FooterSection @isCollapsible={{false}}>
+            {{yield to="social-media"}}
+          </FooterSection>
+        </div>
+        {{#if this.hasHorizontalLine}}
```

Comment:
Could this be named something else, maybe `hasDivider` or just `divided`?

Reply:
Talking with some others, let's go with `hasDivider`

Reply:
I would prefer `hasDivider` as it better hints that it is a boolean